src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -28,6 +28,7 @@ namespace System.Runtime.Intrinsics
     // value instead, thus reducing the number of locals and helping prevent us from hitting
     // the internal inlining limits of the JIT.
 
+    /// <summary>Provides a collection of static convenience methods for creating, manipulating, combining, and converting generic 128-bit vectors.</summary>
```

Comment:
Combining and converting are a small subset of the total operations supported. Maybe generalizing this as the following would be better:
> Provides a collection of static methods for creating, manipulating, and otherwise operating on 128-bit vectors.